src/Controller/CocktailController.php

```diff
@@ -54,7 +68,26 @@ public function submitName(Request $request): Response
 
         return $this->render('cocktail/submit.html.twig', [
             'cocktailForm' => $form->createView(),
-            'result' => $cocktailName->getCocktailName()
+            'result' => $this->cocktailNameRequest($cocktailName)
         ]);
     }
+
+    private function cocktailNameRequest(CocktailName $cocktailName): string
+    {
+        $response = $this->httpClient->request('GET', self::URL, [
```

Comment:
In the next PR make request more pretty: URL constant probably could be defined elsewhere, where all constants for requests are defined (check tutorials).